velox/type/DecimalUtil.h

```diff
@@ -32,5 +32,34 @@ class DecimalUtil {
   /// Helper function to convert a decimal value to string.
   template <typename T>
   static std::string toString(const T& value, const TypePtr& type);
+
+  inline static int128_t rescaleWithRoundUp(
+      const int128_t& unscaledValue,
+      const uint8_t fromScale,
+      const std::pair<int, int>& toPrecisionScale) {
```

Comment:
I would say use `const uint8_t toScale`, `const uint8_t toPrecision` since this is a public API.